packages/beacon-node/src/network/reqresp/request/index.ts

```diff
@@ -109,7 +109,7 @@ export async function sendRequest<T extends IncomingResponseBody | IncomingRespo
     const protocol = protocols.get(protocolId);
     if (!protocol) throw Error(`dialProtocol selected unknown protocolId ${protocolId}`);
 
-    logger.debug("Req  sending request", {...logCtx, body: renderRequestBody(method, requestBody)});
```

Comment:
I think the reason there's an extra space here is to align in the logs with `Resp blah`

Reply:
Yep